docs/howto/how-to-build-runtime-tizen-gbs-rpi4.md

```diff
@@ -174,34 +174,26 @@ $ vi j2/etc/systemd/system/ip.service
 and set as like:
 ```
 [Service]
-Type=simple
 Restart=always
 RestartSec=1
 User=root
-ExecStart=/bin/sh /bin/ip.sh
+ExecStart=/bin/sh -c "ifconfig eth0 192.168.x.y netmask 255.255.255.0 up"
 
 [Install]
 WantedBy=multi-user.target
 ```
+Replace 192.168.x.y to your actual ip address.
 
-(5-3) Add a new file
-```
-$ vi j2/bin/ip.sh
-```
-and set with IP address for your RPi4:
-```
-ifconfig eth0 192.168.x.y netmask 255.255.255.0 up
-```
-where you should update `192.168.x.y` part to your actual IP address.
 
-(5-4) Add a symbolic link
+(5-3) Add a symbolic link
 ```
+$ sudo mkdir -p j2/etc/systemd/system/multi-user.target.wants/
```

Comment:
By default, there is no `multi-user.target.wants`, it must be created explicitly.
For your information,  `multi-user.target.wants` is automatically created if you enable this service with `systemctl enable ip.service`.